rector-migrate.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Transform\Rector\Assign\PropertyFetchToMethodCallRector;
+use Rector\Transform\ValueObject\PropertyFetchToMethodCall;
+
+// This file configures rector/rector to replace all deprecated property usages with their equivalent functions.
+return static function (RectorConfig $rectorConfig): void {
```

Comment:
@OskarStark 

How do you feel about adding a workflow for demonstrating that this configuration works?

Perhaps https://github.com/OskarStark/Faker/pull/1?

Reply:
Make sense